scripts/check-asciidoctor-build.sh

```diff
@@ -42,14 +41,19 @@ check_updated_assemblies () {
         if grep -rq "$PAGE" --include "*.yml" _topic_maps ; then
             # validate the assembly
             echo "Validating $ASSEMBLY ... 🚨"
-            VALIDATION_ERROR=$(asciidoctor "$ASSEMBLY" -a source-highlighter=rouge -a icons! -o /tmp/out.html -v --failure-level WARN --trace)
+            asciidoctor "$ASSEMBLY" -a source-highlighter=rouge -a icons! -o /tmp/out.html -v --failure-level WARN --trace
```

Comment:
**suggestion:** Can we make the errors from this step red for easier identification?
```sh
RED='\033[0;31m'
NC='\033[0m'
output=$(asciidoctor "$ASSEMBLY" -a source-highlighter=rouge -a icons! -o /tmp/out.html -v --failure-level WARN --trace 2>&1)

if [[ $? != 0 ]];
then
    echo -e "${RED}$output${NC}"
    echo "Validation errors found! ❌"
    exit 1
else
    echo "No errors found! ✅"
    exit 0
fi
```